### PR TITLE
Add ClassicLink settings to EC2_launchconfig

### PR DESCRIPTION
**Pull request by evanccnyc**
Fairly simple change that allows for classiclink integration for Launch Configs.

### CODE REVIEW

File: cloud/amazon/ec2_lc.py [hunk 1]
```diff
@@ -116,6 +116,18 @@
     default: false
     aliases: []
     version_added: "1.8"
+  classic_link_vpc_id:
+    description:
+      - Id of ClassicLink enabled VPC
+    required: false
+    default: null
+    version_added: "2.0"
+  classic_link_vpc_security_groups"
```
Comment (gregdek) [inline cloud/amazon/ec2_lc.py]: I'm guessing this should be a colon at the end of classic_link_vpc_security_groups and not a double-quote?

**Comment by evanccnyc** [comment]
Fixed the typo.

**Comment by gregdek** [comment]
@garethr please review; if you give +1, we will promote to "P1_reviewed".

**Comment by garethr** [comment]
@gregdek from a quick look this looks fine to me. Unfortunately I've not been using this code in a little while so I don't have the bandwidth to test out.

**Comment by gregdek** [comment]
Looks safe-ish to me, since it's purely additive; marking priority_reviewed.

**Comment by abadger** [comment]
Thanks!

# Merged

Hi!

This has been merged in, and will also be included in the next major release.

If you or anyone else has any further questions, please let us know by stopping by one of the two mailing lists, as appropriate:
- https://groups.google.com/forum/#!forum/ansible-project - for user questions, tips, and tricks
- https://groups.google.com/forum/#!forum/ansible-devel - for strategy, future planning, and questions about writing code

Because this project is very active, we're unlikely to see comments made on closed tickets, but the mailing list is a great way to ask questions, or post if you don't think this particular
issue is resolved.

Thank you!

**Pull request merged**